src/UserService.Data.Provider/UserService.Data.Provider.csproj

```diff
@@ -2,6 +2,11 @@
 
   <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="5.0.4" />
+    <PackageReference Include="System.ComponentModel.Annotations" Version="5.0.0" />
+  </ItemGroup>
+
```

Comment:
remove

Reply:
done, unfortunately it was created automatically